### DISCUSSION TITLE
complete text

### DISCUSSION THREAD

**Discussion by FELS-Zak** [betalgo/openai#40] [Q&A] [answered]
Hi 
I gee the data back in responsechoices.text but it's truncated - how do i get the complete text ?

**Comment by kayhantolga** [comment, marked as answer]
Hi, Would you mind sharing more context about the issue?

**Reply by FELS-Zak** [reply]
When I increased the tokens I am getting mroe text back as the result 
var completionResult = await openAiService.Completions.CreateCompletion (new CompletionCreateRequest () {
				Prompt = input.Trim(),
				MaxTokens = 1000,
				Temperature = 0.9f
			}, aiModel);
			
			is that the right thing?

**Reply by kayhantolga** [reply]
Yes, If you need longer responses you may need to increase MaxTokens. But don't forget it may increase your billing.

**Reply by FELS-Zak** [reply]
thnx - are you charged in the sandbox?

**Reply by kayhantolga** [reply]
I am not sure about the sandbox, I think yes. 
I am not working for OpenAI. This is an unofficial .NET SDK repository. 
You can ask OpenAI-related questions here: https://community.openai.com/
